main.js

```diff
@@ -117,10 +117,8 @@ function createSavedOutfitCard(outfit) {
             x
             </button>
           </figure>`;
-  var el = document.createElement('div');
-  el.innerHTML = domString;
   var parent = document.querySelector('.saved-cards-container');
-  parent.prepend(el.firstChild);
+  parent.insertAdjacentHTML('afterbegin',domString);
```

Comment:
Hey Erik, Amy just mentioned in her suggestions, that she thought we should change some of the variable names such as domString, el and curId to be more understandable and I think this might be a good pull request to do that in.